ooiui/static/js/views/science/PlottingSelectionView.js

```diff
@@ -291,6 +291,14 @@ var FilterSelectionView = Backbone.View.extend({
   events: {
     'change .selectpicker' : 'onChange'
   },
+  searchIcon: function() {
+    var self = this;
+  $("input[type=text]").css("font-family", "FontAwesome");
```

Comment:
@dlangston i guess my question is why are we passing in the search icon to the partial, and not just define it in the partial? is there a reason? 

instead of `<% searchIcon %>` in the partial, why not `<i class="fa fa-filter"></i>`

also the line `$("input[type=text]").css` is going to change the font family of all input dialogs with type text, seems like a blanket statement, maybe look at using `this.$el.find("input[type=text]").css...`